### PR TITLE
config: Add `plots.out_dir`.

### PR DESCRIPTION
**Pull request by daavoo**
Per https://github.com/iterative/dvc/pull/7911

### CODE REVIEW

File: content/docs/command-reference/config.md [hunk 1]
```diff
@@ -249,10 +249,16 @@ experiments or projects use a similar structure.
...
 
+- `plots.auto_open` - if `true`, DVC will automatically open the HTML file
+  generated by `dvc plots` commands in a browser. `false` by default
+
 - `plots.html_template` - sets a
   [custom HTML template](/doc/command-reference/plots#html-templates) for
   `dvc plots`. Accepts a path relative to the `.dvc/` folder.
 
+- `plots.out_dir` - sets the value to use as default for the `--out` flag of
+  `dvc plots show` / `dvc plots diff`. If not set, `dvc_plots` is used.
```
Comment (jorgeorpinel) [inline content/docs/command-reference/config.md:260]:
```suggestion
- `plots.out_dir` - changes the default value for
  `dvc plots show --outs` and `dvc plots diff --out`. The original default
  value is `dvc_plots`.
```

File: content/docs/command-reference/config.md [hunk 2]
```diff
@@ -249,10 +249,16 @@ experiments or projects use a similar structure.
...
 
+- `plots.auto_open` - if `true`, DVC will automatically open the HTML file
+  generated by `dvc plots` commands in a browser. `false` by default
+
 - `plots.html_template` - sets a
   [custom HTML template](/doc/command-reference/plots#html-templates) for
   `dvc plots`. Accepts a path relative to the `.dvc/` folder.
 
+- `plots.out_dir` - changes the default value for `dvc plots show --outs` and
+  `dvc plots diff --out`. The original default value is `dvc_plots`.
```
Comment (jorgeorpinel) [inline content/docs/command-reference/config.md:260]: Looks like we also need to update the default value info in https://dvc.org/doc/command-reference/plots/show#-o and https://dvc.org/doc/command-reference/plots/diff#-o ?

**Comment by shcheklein** [comment]
@daavoo are we good to merge this now?

File: content/docs/command-reference/plots/diff.md [hunk 3]
```diff
@@ -63,7 +63,8 @@ all the current plots, without comparisons.
   ```
 
 - `-o <path>, --out <path>` - specify a directory to write the HTML file
-  containing the plots (`dvc_plots/` by default).
+  containing the plots. The default is `dvc_plots/` unless
+  [`plots.out_dir`](/doc/command-reference/config#plots) config option is set.
```
Comment (jorgeorpinel) [inline content/docs/command-reference/plots/diff.md:67]:
```suggestion
  containing the plots. The default is `dvc_plots` or the value set with the
  [`plots.out_dir`](/doc/command-reference/config#plots) config option.
```

File: content/docs/command-reference/plots/show.md [hunk 4]
```diff
@@ -37,7 +37,8 @@ please see `dvc plots`.
 ## Options
 
 - `-o <path>, --out <path>` - specify a directory to write the HTML file
-  containing the plots (`dvc_plots/` by default).
+  containing the plots. The default is `dvc_plots/` unless
+  [`plots.out_dir`](/doc/command-reference/config#plots) config option is set.
```
Comment (jorgeorpinel) [inline content/docs/command-reference/plots/show.md:41]:
```suggestion
  containing the plots. The default is `dvc_plots` or the value set with the
  [`plots.out_dir`](/doc/command-reference/config#plots) config option.
```

**Review by jorgeorpinel — COMMENTED** [review]
Committing copy edits:

File: content/docs/command-reference/plots/diff.md [hunk 5]
```diff
@@ -63,7 +63,8 @@ all the current plots, without comparisons.
   ```
 
 - `-o <path>, --out <path>` - specify a directory to write the HTML file
-  containing the plots (`dvc_plots/` by default).
+  containing the plots. The default is `dvc_plots` or the value set with the
+  [`plots.out_dir`](/doc/command-reference/config#plots) config option.
```
Comment (jorgeorpinel) [inline content/docs/command-reference/plots/diff.md:67]: Does it create the path if it doesn't exist?

**Review by jorgeorpinel — APPROVED** [review]

File: content/docs/command-reference/plots/diff.md (hunk 5 above)
Comment (daavoo) [reply content/docs/command-reference/plots/diff.md:67]: Yes

**Pull request merged**